argoexec/rockcraft.yaml

```diff
@@ -1,14 +1,13 @@
-#!/usr/bin/env python3
```

Comment:
Do we know why are we skipping tho whole argo-ui part in this ROCK ? https://github.com/argoproj/argo-workflows/blob/v3.4.16/Dockerfile#L27

Do we know why are we also skipping this part from upstream https://github.com/argoproj/argo-workflows/blob/v3.4.16/Dockerfile#L90

Reply:
Right, that's because they seem to be using a single Dockefile for generating multiple images, that's why they have a multistage build. The argo-ui and the argo-controller would be a completely different image, for this rock we are only interested in the argoexec bits.